flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/api/TableEnvironmentITCase.scala

```diff
@@ -71,9 +75,23 @@ class TableEnvironmentITCase(tableEnvName: String, isStreaming: Boolean) extends
     tableEnvName match {
       case "TableEnvironment" =>
         tEnv = TableEnvironmentImpl.create(settings)
+        if (isStreaming) {
+          tEnv.getConfig.getConfiguration.set(
+            ExecutionCheckpointingOptions.CHECKPOINTING_MODE,
+            CheckpointingMode.EXACTLY_ONCE)
+          tEnv.getConfig.getConfiguration.set(
+            ExecutionCheckpointingOptions.CHECKPOINTING_INTERVAL,
+            Duration.ofMillis(CheckpointCoordinatorConfiguration.MINIMAL_CHECKPOINT_TIME)
+          )
+        }
       case "StreamTableEnvironment" =>
-        tEnv = StreamTableEnvironment.create(
-          StreamExecutionEnvironment.getExecutionEnvironment, settings)
+        val execEnv = StreamExecutionEnvironment.getExecutionEnvironment
+        tEnv = StreamTableEnvironment.create(execEnv, settings)
+        if (isStreaming) {
```

Comment:
also use `tEnv.getConfig.getConfiguration.set()` ? then we can move them into one place